src/pkg/caendr/caendr/services/nemascan_mapping.py

```diff
@@ -2,19 +2,16 @@
 
 from caendr.services.logger import logger
 
-from caendr.models.datastore import NemascanMapping
-from caendr.models.error     import NotFoundError
-from caendr.models.task      import TaskStatus
+from caendr.models.datastore import NemascanReport
 
 from caendr.services.cloud.storage import get_blob_list
-from caendr.services.tools.submit import submit_job
 from caendr.utils.env import get_env_var
 
 NEMASCAN_NXF_CONTAINER_NAME = get_env_var('NEMASCAN_NXF_CONTAINER_NAME', can_be_none=True)
 
 
 
-def is_data_cached(ns: NemascanMapping):
+def is_data_cached(ns: NemascanReport):
```

Comment:
not sure if I misread it earlier, but we should standardize capitalization of `Nemascan` vs `NemaScan`

Reply:
Agreed -- I've changed all instances in the code to `Nemascan`, with lowercase `'s'`.  There are still a few text instances that use `NemaScan`, (e.g. the text on the tool page, env vars & GCP paths), but I'm not sure we can refactor those the same way.